<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replayform</class>
 <widget class="QWidget" name="replayform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replay</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="savePathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="savePathButton">
       <property name="text">
        <string>选择保存路径</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>操作间隔时间（单位/秒）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sleepTimeEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="insertSleepButton">
       <property name="text">
        <string>插入时间</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startRecordButton">
       <property name="text">
        <string>开始录制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopRecordButton">
       <property name="text">
        <string>结束录制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="replayFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="replayFileButton">
       <property name="text">
        <string>选择回放文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>回放次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replayCountEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="startReplayButton">
       <property name="text">
        <string>开始回放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>暂停回放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopReplayButton">
       <property name="text">
        <string>停止回放</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>在拖动的进度位置恢复回放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="logEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>注意：回放拖动测试的APP Icon只能放在launcher第一屏</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="imageLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="windowNameLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliderProgress">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
